Containers/borgbackup/start.sh

```diff
@@ -18,6 +18,9 @@ else
 fi
 export BORG_UNKNOWN_UNENCRYPTED_REPO_ACCESS_IS_OK=yes
 export BORG_RELOCATED_REPO_ACCESS_IS_OK=yes
+if [ -n "$BORG_REMOTE_PATH" ]; then
+    export BORG_REMOTE_PATH
+fi
```

Comment:
I think this can be removed, since the env is not modified here, it does not need to re-exported